internal/fflag/fflag.go

```diff
@@ -0,0 +1,133 @@
+package fflag
```

Comment:
If this package is specifically an implementation of LaunchDarkly then we should just name the package `launchdarkly`.